Makefile

```diff
@@ -472,7 +472,7 @@ nxOMSGenerateInventoryMof:
 
 nxOMSPlugin:
```

Comment:
this module is co-owned by Security and change tracking....Claire chen/Nadav from security and Lahari putrevu, please coordinate releases with themn

Reply:
make sure the other solutions using this module work well in INT too and get sign off from omsagent team as well when you rollout to INT